runtime/port/sysvipc/j9sharedhelper.c

```diff
@@ -293,7 +293,11 @@ cleanSharedMemorySegments(struct J9PortLibrary* portLibrary)
 			return;
 		}
 	}
-#else
+#elif defined(J9ZTPF) /* defined(J9ZOS390) */
+		/* The z/TPF platform does not support the ipcs command */
```

Comment:
This looks to be indented one level too deep

Reply:
Update pushed.